src/hotspot/os/linux/os_linux.cpp

```diff
@@ -223,8 +223,6 @@ julong os::physical_memory() {
       log_trace(os)("total container memory: " JLONG_FORMAT, mem_limit);
       return mem_limit;
     }
-    log_debug(os, container)("container memory limit %s: " JLONG_FORMAT ", using host value",
-                            mem_limit == OSCONTAINER_ERROR ? "failed" : "unlimited", mem_limit);
```

Comment:
It seems the equivalent line in `os::Linux::available_memory()` can get removed too, as we now log this info as part of `OSContainer::memory_limit_in_bytes()`

Reply:
Thanks, done in 40930a81b6a.